kelpie_load_test.py

```
@@ -0,0 +1,422 @@
+        self.start_time = time.time()
+        
+    def measure_tool_execution_real(self):
+        """Measure real tool execution times"""
+        print("📊 Testing actual tool execution times...")
+        
+        tool_tests = []
+        
+        # Test file read operations
+        for i in range(10):
+            start = time.time()
+            result = subprocess.run([
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)